### PR TITLE
Spruce up ephemeral workspace docs

### PR DESCRIPTION
**Pull request by ritsok**
### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/workspaces/settings/deletion.mdx [hunk 1]
```diff
@@ -25,15 +25,33 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
...
+1. Under **Automatically destroy**, click **Set up auto destroy**.
+1. Enter the desired date and time. Terraform Cloud defaults to your local time zone for scheduling and displays how long until the scheduled operation.
+1. Click **Confirm auto destroy**.
+
+To cancel a scheduled auto destroy run in Terraform Cloud:
+1. Navigate to the workspace's **Settings** > **Destruction and Deletion** page.
+1. Under **Automatically destroy**, click **Edit** next to your scheduled run's details.
+1. Click **Remove**.
+
+You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 24 hours before an auto destroy run, and to report auto destroy run results.
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/settings/deletion.mdx:52]:
```suggestion
You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto destroy run, and to report auto destroy run results.
```

File: website/docs/cloud-docs/workspaces/settings/deletion.mdx [hunk 2]
```diff
@@ -25,15 +25,33 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
 @include 'tfc-package-callouts/ephemeral-workspaces.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
-Automated destroy runs are available as part of the ephemeral workspaces feature.
+You can schedule a run to destroy all resources in a workspace at a specific
+date and time. This ephemeral workspace option reduces your infrastructure
```
Comment (judithpatudith) [inline website/docs/cloud-docs/workspaces/settings/deletion.mdx:29]:
```suggestion
date and time. This ephemeral-workspace option reduces your infrastructure
```

**Review by judithpatudith — APPROVED** [review]
Thank you!

File: website/docs/cloud-docs/workspaces/settings/deletion.mdx [hunk 3]
```diff
@@ -25,15 +25,33 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
...
+spend by helping you automatically destroy resources that you only need
+temporarily, such as development environments.
 
-Some use cases may benefit from allowing Terraform Cloud to schedule future infrastructure deletion automatically. For example, you can automatically destroy a workspace's resources used solely in a development environment after a certain period of time to save on cost.
+If you perform a destroy operation in the workspace before the scheduled auto
```
Comment (notchairmk) [inline website/docs/cloud-docs/workspaces/settings/deletion.mdx:33]: This is my bad, the way it should have been written before is more like 

```This setting is automatically unset whenever Terraform Cloud performs an auto destroy run```

So technically not any destroy runs, only the automatic ones will unset this field. The main context here is that users may not anticipate that the setting will automatically be updated once completed, so we wanted to let them know in some way.

File: website/docs/cloud-docs/workspaces/settings/deletion.mdx (hunk 2 above)
Comment (notchairmk) [inline website/docs/cloud-docs/workspaces/settings/deletion.mdx:29]: > This ephemeral workspace option

just for context: Ephemeral Workspaces is an entitlement, which can technically have many features – but right now the only feature under this entitlement is the auto destroy functionality. swapping out `option` -> `entitlement` may be more clear here that it's tied to pricing/subscription. I don't feel strongly about this though, just sprinkling in context.

**Comment by notchairmk** [comment]
really just one comment from me, overall definitely an improvement 😅

File: website/docs/cloud-docs/workspaces/settings/deletion.mdx (hunk 2 above)
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/settings/deletion.mdx:29]:
```suggestion
date and time. This ephemeral workspace entitlement reduces your infrastructure
```

File: website/docs/cloud-docs/workspaces/settings/deletion.mdx (hunk 3 above)
Comment (ritsok) [reply website/docs/cloud-docs/workspaces/settings/deletion.mdx:33]: thanks! i'll clarify

File: website/docs/cloud-docs/workspaces/settings/deletion.mdx [hunk 4]
```diff
@@ -25,15 +25,33 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
...
+destroy, Terraform Cloud will cancel the scheduled run. You can re-enable it to
+schedule another future auto destroy run.
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/settings/deletion.mdx:35]:
```suggestion
After Terraform Cloud performs an auto destroy run, it unsets the `auto-destroy` flag on the workspace. If you continue using the workspace, you can schedule another future auto destroy run to remove any new resources. 
```

**Review by notchairmk — APPROVED** [review]

**Pull request merged**